FluidNC/src/Motors/TMC5160Driver.cpp

```diff
@@ -54,6 +54,10 @@ namespace MotorDrivers {
 
         tmc5160->tpfd(_tpfd);
 
+        tmc5160->diag0_error(_diag0_error);
+        tmc5160->diag0_otpw(_diag0_otpw);
+        tmc5160->diag0_int_pushpull(_diag0_int_pushpull);
```

Comment:
What does the "int" in diag0_int_pushpull signify?  Internal? Integer? Something else?

Reply:
That is the name of the register on the chip. The name is also used in the library.
It apppears to mean internal.

Reply:
I suppose it could also mean "interrupt".